<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCreateOperation</class>
 <widget class="QWidget" name="FormCreateOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание операции</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя операции</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fldOperationName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя образца операции</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbPatternName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать операцию</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ресурсы</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbResourceName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить ресурс</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить ресурс</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
